editor-script-box/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "script-box",
+  "version": "0.1.0",
+  "productName": "Minecraft Script Box Playground",
+  "description": "Minecraft Script Box Playground",
+  "private": true,
+  "devDependencies": {
+    "@minecraft/core-build-tasks": "^1.1.3",
+    "eslint-plugin-minecraft-linting": "^1.2.1",
+    "source-map": "^0.7.4",
+    "ts-node": "^10.9.1",
+    "typescript": "^5.0.2"
+  },
+  "scripts": {
+    "lint": "just-scripts lint",
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "local-deploy": "just-scripts local-deploy",
+    "mcaddon": "just-scripts mcaddon",
+    "enablemcloopback": "CheckNetIsolation.exe LoopbackExempt -a -p=S-1-15-2-1958404141-86561845-1752920682-3514627264-368642714-62675701-733520436",
+    "enablemcpreviewloopback": "CheckNetIsolation.exe LoopbackExempt -a -p=S-1-15-2-424268864-5579737-879501358-346833251-474568803-887069379-4040235476"
+  },
+  "dependencies": {
+    "@minecraft/math": "^1.1.0",
+    "@minecraft/server": "^1.12.0-beta.1.21.0-preview.24",
```

Comment:
Editor basics is using different versions than this one, should they be aligned?

Reply:
I think the idea was to use stable for script-box, but since editor-script-box /must/ be beta APIs then that sample is tied to beta dependencies.  Which is rough because that'll get out of date in no time.